apps/src/storage/dataBrowser/dataVisualizer/CrossTabChart.jsx

```diff
@@ -37,9 +41,40 @@ export default function CrossTabChart(props) {
   //    shrink to fit evenly.
   const columnWidth = 99 / (columns.length - 1) + '%';
 
-  const min = Math.min(...numericValues);
-  const max = Math.max(...numericValues);
+  const {min, max} = numericValues.reduce(
+    (values, currentVal) => {
+      return {
+        min: Math.min(currentVal, values.min),
+        max: Math.max(currentVal, values.max),
+      };
+    },
+    {min: Number.POSITIVE_INFINITY, max: Number.NEGATIVE_INFINITY}
+  );
 
+  if (columns.length > MAX_CROSSTAB_COLUMNS) {
```

Comment:
If these aren't user facing, would it be more useful to log these to our metrics system? What situation would cause these to appear?

Reply:
These are _potentially_ user facing, but they're just an escape hatch in case a user has managed to somehow bypass the filters on which columns they can select and produced a chart that'd crash.

Reply:
And localization _might_ be desired at some point, but we also may want to completely re-do the view, style it, change the behavior, etc. So right now I'm just deferring it as a follow up task for how we want to actually present the error case and then we can try localizing that.

Reply:
I added localization, and arbitrarily tossed the variables into the `common` file, since it looked like the best fit atm.